tests/bdp_util.erl

```diff
@@ -22,7 +22,7 @@
 -module(bdp_util).
 
 -export([build_cluster/1, build_cluster/2,
-         add_service/4, remove_service/2, start_seervice/4, stop_service/4]).
+         add_service/4, remove_service/2, start_service/4, stop_service/4]).
```

Comment:
Let the typo be; fixing it should be done in all modules where it already occurs.